src/main/java/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,70 @@
+package coffee.order;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+public class CoffeeOrderBoard {
+    private Map<Long, Order> orders = new HashMap<Long, Order>();
+    private long numberOfOrders;
+    private static final Logger LOGGER = Logger.getLogger(CoffeeOrderBoard.class.getName());
+    private static final String DEFAULT_PATH_FOR_LOG = "D:/Log.txt";
+    private String pathForLog;
+    private FileHandler fh;
```

Comment:
Do you remember the order of class members? Can you please order them correctly?